xml/System/ReadOnlySpan`1.xml

```diff
@@ -29,9 +29,13 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <typeparam name="T">To be added.</typeparam>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <typeparam name="T">The type of items in the <see cref="System.ReadOnlySpan`1" />.</typeparam>
+    <summary>Provides a type-safe and memory-safe read-only representation of a contiguous region of arbitrary memory.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[`ReadOnlySpan<T>` is a [ref struct](~/docs/csharp/reference-semantics-with-value-types.md#ref-struct-type) that is allocated on the stack rather than on the managed heap. Ref struct types have a number of restrictions to ensure that they cannot be promoted to the managed heap, including that they can't be boxed and they can't be assigned to variables of type <xref:System.Object>, `dynamic`, and to any interface type.
```

Comment:
Instead of "...that is allocated on the stack rather than the managed heap", I'd prefer "must be allocated on the stack and can never escape to the managed heap."  

My reason is that we should start promoting the language in the [draft C# spec](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-7.2/span-safety.md#draft-language-specification) for ref like structs.

Reply:
OK, I updated those sections. Please review to make sure I correctly described it 😄